.github/workflows/license_finder.yml

```diff
@@ -24,4 +24,4 @@ jobs:
 
       - name: Run license finder
         run: |
-          sudo -u testbot bash -lc 'make license-check'
+          sudo -Hu testbot bash -lc 'make license-check'
```

Comment:
-H switches the home directory to the new (testbot) user. This is the default for the sudo in the previous canon images, but it wasn't guaranteed, so making it explicit (don't want to waste the caching.)